assets/src/application/ui/input/Button/Button.tsx

```diff
@@ -17,28 +17,19 @@ type BtnType = React.ComponentType<ButtonProps>;
  */
 const Button = React.forwardRef<typeof ButtonAdapter, ButtonProps>(
 	(
-		{
-			buttonSize = ButtonSize.DEFAULT,
-			buttonText,
-			buttonType = ButtonType.DEFAULT,
-			className: htmlClass,
-			icon,
-			onClick,
-			...props
-		},
+		{ buttonSize = ButtonSize.DEFAULT, buttonText, buttonType = ButtonType.DEFAULT, icon, onClick, ...props },
 		ref
 	) => {
-		const className = classNames('ee-button', {
-			[htmlClass]: htmlClass,
+		const className = classNames('ee-btn-base ee-btn', props.className, {
 			'ee-btn-accent': buttonType === ButtonType.ACCENT,
+			'ee-btn-big': buttonSize === ButtonSize.BIG,
 			'ee-btn-default': buttonType === ButtonType.DEFAULT,
+			'ee-btn-huge': buttonSize === ButtonSize.HUGE,
 			'ee-btn-primary': buttonType === ButtonType.PRIMARY,
 			'ee-btn-minimal': buttonType === ButtonType.MINIMAL,
 			'ee-btn-secondary': buttonType === ButtonType.SECONDARY,
-			'ee-btn-tiny': buttonSize === ButtonSize.TINY,
 			'ee-btn-small': buttonSize === ButtonSize.SMALL,
-			'ee-btn-big': buttonSize === ButtonSize.BIG,
-			'ee-btn-huge': buttonSize === ButtonSize.HUGE,
+			'ee-btn-tiny': buttonSize === ButtonSize.TINY,
```

Comment:
i get that you want to alphabetize things but imho it is worse to break up the logical groupings, which makes it more difficult to differentiate `ButtonType`s from `ButtonSize`es.

That said, what would have been better is if I made the values for those enums lowercase so that we could use them directly to generate these classes, ie:

```ts
const className = classNames('ee-btn-base ee-btn', `ee-btn--${buttonSize}`, `ee-btn--${buttonType}`, props.className);
```

what do you think about doing that instead?